sources/platform/actors/development/actor_definition/output_schema/index.md

```diff
@@ -0,0 +1,296 @@
+---
+title: Actor output schema
+sidebar_label: Actor output schema
+sidebar_position: 5
+description: Learn how to define and present output of your Actor.
+slug: /actors/development/actor-definition/output-schema
+---
+
+**Learn how to define and present the output of your Actor.**
+
+---
+
+The Actor output schema builds upon the schemas for the dataset and key-value store, and defines where the output of an Actor run is stored. It allows you to define templates for URLs where users can find the output generated by your Actor. The outputs defined in the schema are used in the UI to manage how the output is displayed and are returned in the Actor run's `GET` endpoint for automated systems using the API.
```

Comment:
What about adding one sentence that describes the benefits for integrations (good output schema = simpler integration with other actors and tools)?